rcjaRegistration/templates/events/dashboard.html

```diff
@@ -123,4 +123,40 @@ <h1>Your Events:</h1>
     </div>
 
 </div>
+
+<div class="ui modal firstTimePopup">
+    <div class="header">
+        Would you like to sign up for Association membership?
```

Comment:
@evanbailey could you please take a look at this wording and edit as you would like? Can also change the text and colour of the buttons.